numpy/ma/tests/test_core.py

```diff
@@ -4826,6 +4826,16 @@ def test_copy(self):
             np.ma.masked.copy() is np.ma.masked,
             np.True_.copy() is np.True_)
 
+    def test__copy(self):
+        assert_equal(
+            copy.copy(np.ma.masked) is np.ma.masked,
+            copy.copy(np.True_) is np.True_)
```

Comment:
Not incorrect, but I'd probably go for the simpler
```
assert_(copy.copy(np.ma.masked) is np.ma.masked)
```
(note the underscore)

Reply:
I don't believe this - the code you copied this from was code that I wrote, and I disagreed with it!